frontend/package.json

```diff
@@ -16,19 +16,21 @@
     "grunt-contrib-watch": "^1.0.0",
     "grunt-filerev": "^2.3.1",
     "grunt-include-source": "^1.0.0",
+    "grunt-ng-annotate": "^3.0.0",
     "grunt-usemin": "^3.1.1",
     "grunt-wiredep": "^3.0.1",
     "http-server": "^0.9.0",
```

Comment:
I have uninstalled the http-server node package, and have found out, that it was a unnessesary package. So we use the spa-http-server now, that includes push-state for spa